docroot/modules/humsci/hs_layouts/layouts/three-column/three-column.html.twig

```diff
@@ -6,21 +6,25 @@
 
 {% set attributes = attributes.addClass([settings.section_width, 'clearfix', 'three-column']) %}
 
-<div{{ attributes }}>
-  {% if content.left_sidebar %}
-    <div class="decanter-width-one-fourth region">
-      {{ content.left_sidebar }}
-    </div>
-  {% endif %}
+{% if content.main or content.left_sidebar or content.right_sidebar %}
```

Comment:
This check seems redundant. Are you likely to not have anything at all? If so, why use the template?